docs/docs/tutorials/orca_k8s/orca_k8s_tutorial.md

```diff
@@ -0,0 +1,563 @@
+
+__Note:__
+* The __Client Container__ contains all the required environment and libraries except Hadoop/K8s configs.
+* You needn't to create an __Executor Container__ manually, which is scheduled by K8s at runtime.
+
+
+We __highly recommand__ you to specify more arguments when using `spark-submit` script:
+```bash
+sudo docker run -itd --net=host \
+    -v /etc/kubernetes:/etc/kubernetes \
+    -v /root/.kube:/root/.kube \
+    -v /path/to/nfsdata:/bigdl/data \
+    -e NOTEBOOK_PORT=12345 \
+    -e NOTEBOOK_TOKEN="your-token" \
+    -e http_proxy=http://your-proxy-host:your-proxy-port \
+    -e https_proxy=https://your-proxy-host:your-proxy-port \
+    -e RUNTIME_SPARK_MASTER=k8s://https://<k8s-apiserver-host>:<k8s-apiserver-port> \
+    -e RUNTIME_K8S_SERVICE_ACCOUNT=spark \
+    -e RUNTIME_K8S_SPARK_IMAGE=intelanalytics/bigdl-k8s:latest \
+    -e RUNTIME_PERSISTENT_VOLUME_CLAIM=nfsvolumeclaim \
+    -e RUNTIME_DRIVER_HOST=x.x.x.x \
+    -e RUNTIME_DRIVER_PORT=54321 \
+    -e RUNTIME_EXECUTOR_INSTANCES=2 \
+    -e RUNTIME_EXECUTOR_CORES=2 \
+    -e RUNTIME_EXECUTOR_MEMORY=20g \
+    -e RUNTIME_TOTAL_EXECUTOR_CORES=4 \
+    -e RUNTIME_DRIVER_CORES=4 \
+    -e RUNTIME_DRIVER_MEMORY=10g \
+    intelanalytics/bigdl-k8s:latest bash 
+```
+
+In the script:
+* `--net=host`: use the host network stack for the Docker container;
+* `/etc/kubernetes:/etc/kubernetes`: specify the path of kubernetes configurations;
+* `/root/.kube:/root/.kube`: specify the path of kubernetes installation;
+* `/path/to/nfsdata:/bigdl/data`: mount NFS path on host in container as the sepcified path in value; 
+* `NOTEBOOK_PORT`: an Integer that specifies port number for Notebook (only required by notebook);
+* `NOTEBOOK_TOKEN`: a String that specifies the token for Notebook (only required by notebook);
+* `RUNTIME_SPARK_MASTER`: a URL format that specifies the Spark master;
+* `RUNTIME_K8S_SERVICE_ACCOUNT`: a String that specifies the service account for driver pod;
+* `RUNTIME_K8S_SPARK_IMAGE`: the lanuched k8s image;
+* `RUNTIME_PERSISTENT_VOLUME_CLAIM`: a String that specifies the Kubernetes volumeName;
+* `RUNTIME_DRIVER_HOST`: a URL format that specifies the driver localhost (only required by client mode);
+* `RUNTIME_DRIVER_PORT`: a String that specifies the driver port (only required by client mode);
+* `RUNTIME_EXECUTOR_INSTANCES`: an Integer that specifies the number of executors;
+* `RUNTIME_EXECUTOR_CORES`: an Integer that specifies the number of cores for each executor;
+* `RUNTIME_EXECUTOR_MEMORY`: a String that specifies the memory for each executor;
+* `RUNTIME_TOTAL_EXECUTOR_CORES`: an Integer that specifies the number of cores for all executors;
+* `RUNTIME_DRIVER_CORES`: an Integer that specifies the number of cores for the driver node;
+* `RUNTIME_DRIVER_MEMORY`: a String that specifies the memory for the driver node;
+
+## 2.3 Launch the K8s Client Container
+Once the container is created, docker image would return the <containerID>.
```

Comment:
![image](https://user-images.githubusercontent.com/21994046/189259015-5f61af6e-e451-4335-88d5-8b6f6cced85f.png)
containerId not rendering well